src/FastSerialization/GrowableArray.cs

```diff
@@ -399,15 +399,20 @@ private void Realloc(int minSize)
             }
             else
             {
-                int expandSize = array.Length * 3 / 2 + 8;
-                if (minSize < expandSize)
+                long expandSize = (long)array.Length * 3 / 2 + 8;
+                if (minSize > expandSize)
```

Comment:
Would you mind reverting `>` back to `<`?  Otherwise, this change looks good to me.

Reply:
Update to closer match the original